dev/plot_film_parameters.py

```diff
@@ -101,7 +102,7 @@ def visualize_tsne(data, metadata_values, layer_no, fname_out):
 
     tsne_df = pd.concat([tsne_df, tsne_df2])
 
-    print('t-SNE done!')
+    logger.info('t-SNE done!')
```

Comment:
Some of the print statements use a single quote for the string whereas others use double quote. You might want to use a code formatter (like [black](https://black.readthedocs.io/en/stable/)) that will unify them.